.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jacksayshi @alar12
```

Comment:
This doesn't seem like the change your PR should be making

Reply:
I had to make this change to fix merge conflicts with the repo. We attempted walking back my branch and then pulling the repo again, but it would never pull the CODEOWNERS file, and since it wouldn't, I could never push my code to the repo. We fixed it by making my own CODEOWNERS file just as it was in the repo. Sorry for not clarifying this when I first created this pull request.